Library/Formula/trafficserver.rb

```diff
@@ -39,17 +40,26 @@ def install
     # Fix lib/perl/Makefile.pl failing with:
     # Only one of PREFIX or INSTALL_BASE can be given.  Not both.
     ENV.delete "PERL_MM_OPT"
-    system "autoreconf", "-fvi" if build.head?
-    args = [
-      "--prefix=#{prefix}",
-      "--mandir=#{man}",
-      "--with-openssl=#{Formula["openssl"].opt_prefix}",
-      "--with-user=#{ENV["USER"]}",
-      "--with-group=admin"
+
+    (var/"log/trafficserver").mkpath
+    (var/"trafficserver").mkpath
+
+    args = %W[
+      --prefix=#{prefix}
+      --mandir=#{man}
+      --localstatedir=#{var}
+      --sysconfdir=#{etc}
+      --with-openssl=#{Formula["openssl"].opt_prefix}
+      --with-user=#{ENV["USER"]}
```

Comment:
Is the user variable hard coded? we may want to set it during post_install for bottle.